glide.lock

```diff
@@ -392,7 +392,7 @@ imports:
   subpackages:
   - log
 - name: github.com/emicklei/go-restful-swagger12
-  version: 76d80a40857969dddc1a4c21d1eb82f7ea20e9f5
+  version: 5e28dc7157ebd9247a268c26dd1bee7818aa33f7
```

Comment:
looks like this is undoing a bad update from https://github.com/openshift/origin/pull/19494/commits/1c853bdf4bf2e47ba6be53e18d38a95663fe9847 ... @smarterclayton any idea what happened on your PR? 76d80a40857969dddc1a4c21d1eb82f7ea20e9f5 doesn't exist, and 5e28dc7157ebd9247a268c26dd1bee7818aa33f7 is correct